cli/dcoscli/main.py

```diff
@@ -89,43 +91,57 @@ def _main():
     subproc = Popen([executable,  command] + args['<args>'],
                     stderr=PIPE)
 
+    if log_level:
```

Comment:
persist ie keep logging with that level for all commands until they unset the env variable